src/hooks/useTokenLauncher.ts

```diff
@@ -15,15 +15,10 @@ TL.configure(config);
 export function useTokenLauncher(): typeof TokenLauncher {
   const { getTokenLauncherSupportedChainInfo } = useBackendNetworksStore();
   const launcherSupportedChains = getTokenLauncherSupportedChainInfo();
-
-  const currentConfig = useMemo(() => TokenLauncher.getConfig(), []);
-
   useEffect(() => {
-    if (currentConfig.SUPPORTED_NETWORKS?.length === 0) {
-      TL.configure({
-        SUPPORTED_NETWORKS: launcherSupportedChains,
-      });
-    }
+    TL.configure({
+      SUPPORTED_NETWORKS: launcherSupportedChains,
+    });
```

Comment:
unrelated to the issue fixed in this PR, but simplified this in case we run into a scenario where networks happened to update while a session is occurring.